### DISCUSSION TITLE
Access to documents? Photos?

### DISCUSSION THREAD

**Discussion by imbwaldo** [depcheck/depcheck#644] [Q&A]
I installed depcheck on my MacBook and ran it, but was put off a bit when I was asked if it could have access to Documents … and Photos. I declined. Was wondering why depcheck would need that access – especially to Photos. I'm guessing there's a reasonable explanation, but I lack the imagination to come up with it on my own. Could someone please fill in that blank for me?

Thanks!

**Comment by rumpl** [comment]
Hi @imbwaldo 

depcheck shouldn't need access to your Documents or Photos, that is really odd. In what directory were you when you tried to run it?

**Reply by imbwaldo** [reply]
Thanks, Djordje! Sorry for my slow reply.

I believe I was in my root directory. That would explain things, wouldn't it? I just tried it again in a project directory and it worked fine. I should've tried that earlier, but I must confess I got a bit spooked. :)

Thanks very much. Take it easy …
